kotlinx-coroutines-test/common/src/TestDispatcher.kt

```diff
@@ -10,6 +10,9 @@ import kotlin.jvm.*
 
 /**
  * A test dispatcher that can interface with a [TestCoroutineScheduler].
+ * 
+ * @see StandardTestDispatcher A predictable, controllable dispatcher.
+ * @see UnconfinedTestDispatcher A dispatcher that is not confined to any particular thread.
```

Comment:
```suggestion
 * The available implementations are:
 * * [StandardTestDispatcher] is a dispatcher that places new tasks into a queue.
 * * [UnconfinedTestDispatcher] is a dispatcher that behaves like [Dispatchers.Unconfined] while allowing to control
 *   the virtual time.
```
Two things here:
* The description after `@see` doesn't seem to have an effect. 
![2022-03-17-112319_794x207_scrot](https://user-images.githubusercontent.com/52952525/158767296-0a31809f-545b-4dcd-b340-131af80e11ed.png)
* I think not being confined to a particular thread is not the most important property of `UnconfinedTestDispatcher`, as tests are typically single-threaded. Forming an event loop is the useful property here. However, describing the event loop here would, in my opinion, be excessive, so, instead, `Dispatchers.Unconfined` is referenced, which explains this.

What do you think?